src/core/qgsjsonutils.h

```diff
@@ -198,6 +199,19 @@ class CORE_EXPORT QgsJsonExporter
                            const QVariantMap &extraProperties = QVariantMap(),
                            const QVariant &id = QVariant() ) const;
 
+    /**
+     * Returns a GeoJSON string representation of a feature.
```

Comment:
Is it returning a string or a QJsonObject (or is the QJSonObject a string)?

Reply:
@m-kuhn please wait, this is just POC, I didn't fix all docstrings and tests are missing, I'm just looking for general feedback on the idea.

Reply:
@elpaso all good, no hurry. There are two reasons that make me comment (likely to be continued) POC's anyway: 1. once I look at the code and see things, I don't want to care for it a second time. 2. Sometimes POC's are suddenly "accepted and merged" in less than 24 h.

Reply:
This one is falling down in my TODO list btw ...